src/website/src/app/documentation/demos/toggles/toggles.demo.html

```diff
@@ -4,51 +4,320 @@
   ~ The full license information can be found in LICENSE in the root directory of this project.
   -->
 <clr-doc-wrapper [ng]="ng" [ui]="ui" [title]="title" [newLayout]="newLayout">
+  <article>
+    <h5 class="component-summary" id="summary">
+      With toggle switches, users can select the one or off state of an option.
+    </h5>
 
-    <article>
-        <h5 class="component-summary" id="a-toggle-switch-changes-on-and-off-states-of-a-single-setting">A
-            toggle switch changes on and off states of a single setting.</h5>
+    <h2 id="forms-code-samples">Angular Toggle Switch Component</h2>
 
-        <div id="code-examples">
-            <p>To create a toggle switch, wrap an <code class="clr-code">input[type="checkbox"]</code>
-                element and a <code class="clr-code">label</code> element in a block-level wrapper
-                (like a <code class="clr-code">div</code>) with the <code class="clr-code">.toggle-switch</code>
-                classname. To use a toggle without any label, keep the <code class="clr-code">label</code>
-                tag empty. The default toggle is gray in its “off” state and green in
-                its “on” state. The user can change states by clicking on either the toggle or its
-                accompanying label.  You can move the label to the right side by adding the <code class="clr-code">right-label</code> class.</p>
+    <p>If you are using Angular, the recommended approach is to always use the <code class="clr-code">ClrToggle</code>
+      directive on your toggle switches to help manage the form control. By using <code class="clr-code">ClrToggle</code>,
+      you'll automatically be able to leverage built in validation, helper text, and layout features in forms.</p>
 
-            <h3 id="disabled-toggles">Disabled Toggles</h3>
+    <h4>Basic checkbox button</h4>
 
-            <p>To disable a toggle switch, add the <code class="clr-code">disabled</code> attribute to the
-                toggle’s <code class="clr-code">checkbox</code> input. When disabled, the toggle appears
-                gray. The user cannot change the state of the toggle. And the present state is reflected
-                in both the location of the toggle switch and by the “on” state being a slightly darker
-                gray.</p>
+    <p>While it is possible to create a toggle switch without a label, the cases are rare. However, it is possible
+      by just adding <code class="clr-code">clrToggle</code> to your checkbox button to wire up the
+      directive. Notice, it is necessary to add <code class="clr-code">type="checkbox"</code>.</p>
 
-            <clr-toggles-example-demo></clr-toggles-example-demo>
+    <p></p>
+
+    <input type="checkbox" clrToggle />
+
+    <p></p>
```

Comment:
I'm pretty sure that for proper spacing, you should simply do `<p><input type="checkbox" clrToggle /></p>` and remove the empty paragraphs before and after. That's how you would do it in a semantic way in an app anyway.

Reply:
There are a lot of `<p>` because the examples sometimes sit too close to the text, and for now this has been done in a few other places. It would really need a review at the docs level.